src/collective/solr/profiles/default/types.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
```

Comment:
@reinhardt collective.solr does not require plone.app.contenttypes. We still have the Archetypes extender stuff in the package. Does that work with a non p.a.contenttype site and not break the type definitions?